package.json

```diff
@@ -8,7 +8,7 @@
     "preinstall": "./scripts/check_gm.sh",
     "shrink": "npmshrink",
     "start": "scripts/run_dev.js",
-    "test": "grunt test --node-env=test"
+    "test": "scripts/test-local.sh"
```

Comment:
This script, and the `mocha-coverage.js` script that it calls, were copied straight from oauth-server.